src/platform/testing/e2e/nightwatch-commands/allyCheckFocusableCount.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable func-names, prefer-arrow-callback */
+
+const ally = require('ally.js');
+
+/**
+ * Checks for the number of focusable elements on the page,
+ * using the ally.query.tabbable method:
+ * https://allyjs.io/api/query/focusable.html
+ *
+ * "Focusable" means the element or elements can receive
+ * keyboard focus natively by pressing `TAB` or `SHIFT + TAB`,
+ * and elements that have a tabindex="-1" attribute, and can be
+ * focused programmatically.
+ *
+ * @method allyCheckFocusableCount
+ * @param {string} [selector] The selector (CSS / Xpath) used to locate the element.
+ * @param {function} [callback] Optional callback function to be called when the command finishes.
+ * @returns {array} The array of focusable elements found in the selector.
+ * @api commands
+ */
+exports.command = function allyCheckFocusableCount(selector, callback) {
```

Comment:
Is this command useful outside the context of the assertion? If not, consider moving inside the assertion function.